examples/source/1.components/amp-bind-macro.html

```diff
@@ -0,0 +1,57 @@
+<!---
+
+author: elisameyer, philkrie
+
+--->
+<!--
+  ## Introduction
+
+  This example shows how you can sanitize user input for a URL with an
+  `amp-bind-macro` call.
+-->
+<!doctype html>
+<html ⚡="">
+  <head>
+    <title>amp-bind-macro</title>
+    <meta charset="utf-8" />
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <link rel="canonical" href="<% canonical %>">
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <!-- ## Setup -->
+    <!-- Import the `amp-bind` component to use `amp-bind-macro` -->
+    <script
+      async
+      custom-element="amp-bind"
+      src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"
+    ></script>
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  </head>
+  <body>
+    <h1>
+      AMP bind macro text sanitization
+    </h1>
+    <!-- ## Implementation -->
+    <!-- the `replaceSpace` expression defined in `amp-bind-macro` replaces spaces in user input with %20
+          and sets all letters to lowercase to match url format for google search -->
+    <amp-bind-macro
```

Comment:
can you wrap <amp-bind-macro> until to the last <p> in an empty div. This way it'll will show up as a consistent code snippet and will also work in the doc preview. Similar to here:

https://amp.dev/documentation/examples/interactivity-dynamic-content/restrict_duplicate_selection/?format=websites#setup

Reply:
updated